lib/v2/screens/app/interactor/usecases/guardians_notification_use_case.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:seeds/providers/services/firebase/firebase_database_service.dart';
+import 'package:seeds/v2/datasource/local/settings_storage.dart';
+
+class GuardiansNotificationUseCase {
+  Stream<bool?> get hasGuardianNotificationPending {
+    return FirebaseDatabaseService().hasGuardianNotificationPending(settingsStorage.accountName);
```

Comment:
This is calling a v1 Service. 
We dont call v1 services from V2. 

Please move this service call to v2 before calling it.

Reply:
Go it can i do it in a new PR?

Reply:
yea, thats cool